runtime/vm/resolvesupport.cpp

```diff
@@ -1367,13 +1367,13 @@ resolveVirtualMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
 		 * and storing the result into the cpClass->methodTypes table.
 		 */
 		if ((NULL != cpShapeDescription)
-		&& (resolvedClass == J9VMJAVALANGINVOKEMETHODHANDLE(vm))
+			&& (resolvedClass == J9VMJAVALANGINVOKEMETHODHANDLE(vm))
```

Comment:
I'd prefer that these spacing changes not be made.  The formatting without it clear that the the if statement continues as a blcok